src/Main.cc

```diff
@@ -491,10 +491,29 @@ static void showUsage(const char* prog, const char* init_filename)
 
 static void showSplash()
 {
+#ifdef ENABLE_MPL2
+  bool enable_mpl2 = true;
+#else
+  bool enable_mpl2 = false;
+#endif
+#ifdef ENABLE_PAR
+  bool enable_par = true;
+#else
+  bool enable_par = false;
+#endif
```

Comment:
@rovinski Not sure if this is the best way to do it but we can repeat the same for the other headers too were we to avoid creating new cmake generated files

Reply:
I think you might be able to simplify this with a macro. Also use `static constexpr`
Something like:
```
#define IS_COMPILED(DEF,VAR) \
  #ifdef DEF \
    static constexpr VAR = true; \
  #else \
    static constexpr VAR = false; \
  #endif

IS_COMPILED(ENABLE_PAR,enable_par)
IS_COMPILED(ENABLE_MPL2,enable_mpl2)
...
```